WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsJetpackSocialContainerView.kt

```diff
@@ -0,0 +1,121 @@
+package org.wordpress.android.ui.posts
+
+import android.content.res.Configuration
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Divider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import org.wordpress.android.models.PublicizeConnection
+import org.wordpress.android.models.PublicizeConnectionList
+import org.wordpress.android.ui.compose.components.TrainOfIconsModel
+import org.wordpress.android.ui.compose.components.buttons.PrimaryButton
+import org.wordpress.android.ui.compose.theme.AppThemeEditor
+import org.wordpress.android.ui.compose.unit.Margin
+import org.wordpress.android.ui.posts.social.PostSocialConnection
+import org.wordpress.android.ui.posts.social.compose.PostSocialMessageItem
+import org.wordpress.android.ui.posts.social.compose.PostSocialSharesText
+import org.wordpress.android.ui.publicize.PublicizeServiceIcon
+
+@Composable
+fun EditPostSettingsJetpackSocialContainerView(
```

Comment:
⚠️  I would avoid using the suffix `View` for Composables as it might be confused with View-based UIs/components, which usually have that suffix (e.g.: `TextView`, `ImageView`, etc). Also, since we are using both XMLs (view-based) and Compose in this codebase I would reserve the View suffix for View-wrappers around Composables if needed.

In this case, the file names are really confusing because the `EditPostSettingsJetpackSocialContainerComposeView` has `Compose` in the name but it is NOT a Composable, while this houses a Composable but that has `View` in the name.

I suggest renaming this component to `EditPostSettingsJetpackSocialContainer`, and renaming the view wrapper to `EditPostSettingsJetpackSocialContainerView`.

Reply:
Good catch, thanks!